public/components/FeatureAnywhereContextMenu/CreateAnomalyDetector/CreateNew/CreateNew.js

```diff
@@ -0,0 +1,474 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React, { useState } from 'react';
+import { 
+  EuiTitle, 
+  EuiSpacer, 
+  EuiIcon, 
+  EuiText, 
+  EuiSwitch, 
+  EuiLoadingSpinner,
+  EuiPanel,
+  EuiAccordion,
+  EuiFormRow,
+  EuiFieldText,
+  EuiCheckbox,
+  EuiSelect,
+  EuiFlexItem,
+  EuiFlexGroup,
+  EuiFieldNumber,
+  EuiCallOut
+ } from '@elastic/eui';
+import { EmbeddablePanel } from '../../../../../../../src/plugins/embeddable/public';
+import './styles.scss';
+import EnhancedAccordion from '../../EnhancedAccordion';
+
+function CreateNew({ embeddable, closeFlyout, core, services, index }) {
+  const [isShowVis, setIsShowVis] = useState(false);
+  const title = embeddable.getTitle();
+  const history = {
+    location: { pathname: '/create-detector', search: '', hash: '', state: undefined },
+    push: (value) => console.log('pushed', value),
```

Comment:
can remove log line. is the use of `push` going to be needed, or can we leave to be whatever the default is?